src/types/account-members.d.ts

```diff
@@ -14,6 +14,10 @@ export interface AccountMemberBase extends Identifiable {
     email?: string
 }
 
+export interface AccountMemberCustom extends AccountMemberBase {
```

Comment:
There is no need for this type. `Update()` should be just accepting `Partial<AccountMember>` type. Customers with custom flow fields would then have two options: 1. cast their object to any when calling Update() and 2. augumenting sdk's AccountMember type.